refinery/data_set_manager/test_utils.py

```diff
@@ -448,11 +449,11 @@ def test_is_field_in_hidden_list(self):
     def test_generate_solr_params_no_params_returns_obj(self):
         # empty params
         query = generate_solr_params_for_assay(QueryDict({}), self.valid_uuid)
-        self.assertItemsEqual(sorted(query.keys()), ['json', 'params'])
+        self.assertCountEqual(sorted(query.keys()), ['json', 'params'])
```

Comment:
This looks like a much weaker test. Is there anything similar to `assertItemsEqual`?

Reply:
my understanding is that they are identical - i have a link to a reference in the issue for this but here is another: https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertItemsEqual